openpower/configs/hostboot/mihawk.config

```diff
@@ -0,0 +1,73 @@
+# The BMC MBOX Protocol is used to access PNOR
+unset SFC_IS_AST2500
+unset SFC_IS_AST2400
+set   PNORDD_IS_BMCMBOX
```

Comment:
I'm pretty sure we want this being PNORDD_IS_IPMI. Is this right @amboar / @dcrowell77 ?

Reply:
Yes. PNORDD_IS_BMCMBOX is deprecated. Do we have a good way of indicating that?

Reply:
Not really, we don't have a "openpower template" config file, though that is a good idea now that I say that out loud... ;-)

Reply:
And witherspoon.config still has PNORDD_IS_BMCMBOX set.  However, our code is actually written to allow either mode dynamically.  That was done to handle the non-crisp transition.  

So, the summary is - this is fine.

Reply:
Yep, it's fine in that it will use try IPMI anyway, but it is a bit confusing. There's no reason not to switch all the P9 machines at this point. We shouldn't be adding new machines with a confusing config option

Reply:
We're follow the setting in Witherspoon and Zaius. Can we use PNORDD_IS_BMCMBOX currently? Until other P9 machines switch to IPMI, we'll follow them.

Reply:
Yes, the use of this is acceptable. Agree we should have the other P9 machines change first.